go/processor/floria/processor.go

```diff
@@ -93,6 +112,14 @@ func (p *processor) Run(
 	}, nil
 }
 
+type runContext struct {
+	tosca.TransactionContext
+	interpreter           tosca.Interpreter
+	blockParams           tosca.BlockParameters
+	transactionParameters tosca.TransactionParameters
```

Comment:
nit: would be nice to be consistent ... either both are `Params` or `Parameters`